src/logic/queryParams.js

```diff
@@ -35,7 +36,22 @@ import {
  * @return {string}
  */
 const getSearchQueryFromQueryParameterOrLocalStorage = route => {
-  const { [CommonQueryParameters.SearchFilters]: sq } = route?.query;
+  const {
+    [CommonQueryParameters.SearchFilters]: sq,
+    [CommonQueryParameters.PlainSearchFilters]: f,
+  } = route?.query;
+
+  // parse json then serialize
+  if (typeof f === 'string') {
+    console.info('getSearchQueryFromQueryParameterOrLocalStorage', f);
+    try {
+      return serializeFilters(JSON.parse(f));
+    } catch (err) {
+      console.error(err);
```

Comment:
Can we remove `console` here?

Reply:
well I prefer to see the error if there's any

Reply:
You probably wanted to see a warning (`console.warn`). Logging an error and continuing a normal flow is confusing.

Reply:
ok